src/app/helpers/tenantHelper.js

```diff
@@ -15,28 +15,37 @@ const successResponseStatusCode = 200
 module.exports = {
 
   getImagePath: function (baseUrl, tenantId, image, callback) {
+    let cbLocalTenant = true;
     fs.stat(path.join(__dirname, '../tenant', tenantId, image), function (err, stat) {
       if (err) {
         if (envHelper.DEFAULT_CHANNEL && _.isString(envHelper.DEFAULT_CHANNEL)) {
           fs.stat(path.join(__dirname, '../tenant', envHelper.DEFAULT_CHANNEL, image), function (error, stat) {
             if (error) {
-              callback(null, null)
+              module.exports.checkTenantCdnUrl(baseUrl, tenantId, image, '' ,callback)
```

Comment:
please send false in place of empty string

Reply:
yes done